### PR TITLE
Initial documentation for APIM

### LINKED ISSUE #556: Document use of APIM
Documentation should include:
- Configuration details for our specific use case
- Process for making API requests
- Process for issuing and managing subscription keys
- Explanation for giving a subscription access to an API operation

### PR DESCRIPTION
**Pull request by kingcomma**
Closes #556 and final step of #28.

### CODE REVIEW

File: match/docs/public-api.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Public API
```
Comment (ryanhofdotgov) [inline match/docs/public-api.md:1]: thought: we probably need to think about this nomenclature as this API isn't really public. *External* is more accurate, but not very precise.

File: match/docs/public-api.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- [Operations](https://docs.microsoft.com/en-us/azure/api-management/api-management-key-concepts#-apis-and-operations)
+- [Policies](https://docs.microsoft.com/en-us/azure/api-management/api-management-howto-policies)
+- [Versions](https://docs.microsoft.com/en-us/azure/api-management/api-management-versions)
+- [Subscriptions](https://docs.microsoft.com/en-us/azure/api-management/api-management-subscriptions)
+
+## Calling the API
+
+To call an endpoint:
+
+1. [Procure an API key](#managing-api-keys)
```
Comment (ryanhofdotgov) [inline match/docs/public-api.md:21]:
```suggestion
1. [Obtain an API key](#managing-api-keys)
```
"Procure" in government usually implies some type of purchasing process.

**Review by ryanhofdotgov — APPROVED** [review]
LGTM – we can come back to the nomenclature. Perhaps a topic for our sync on Friday.

File: match/docs/public-api.md (hunk 1 above)
Comment (kingcomma) [reply match/docs/public-api.md:1]: Agree :100: I think the weekly sync is a great place to discuss.

**Pull request merged**